relay-general/src/protocol/debugmeta.rs

```diff
@@ -486,6 +508,8 @@ pub enum DebugImage {
     Wasm(Box<NativeDebugImage>),
     /// Source map debug image.
     SourceMap(Box<SourceMapDebugImage>),
+    /// JVM based debug image.
+    JvmBased(Box<JvmBasedDebugImage>),
```

Comment:
I would rename `JvmBased` to `Jvm` everywhere, but that is just a personal preference.